mangaki/mangaki/management/commands/add_anidb.py

```diff
@@ -1,11 +1,15 @@
 from django.core.management.base import BaseCommand
 from mangaki.utils.anidb import AniDB
-from mangaki.models import Work, Category
-from urllib.parse import urlparse
+from mangaki.models import WorkTitle
+
+
+def create_anime(work, work_titles):
```

Comment:
This changed behavior. It would previously update an existing anime with the given anidb_aid while it now creates a new anime with same anidb_aid. We are currently trying to remove duplicates from our database, not introduce new ways of creating them ;-)

Besides, it looks like this is only ever used with the return value from `Anidb.get_mangaki_work`. Why not simplify things and make `get_mangaki_work` return the corresponding `Work` object instead (which would also be much more coherent with its name)? The `update_or_create` logic should be reintroduced there, of course.